include/miopen/config.h.in

```diff
@@ -78,11 +78,62 @@
 
 // clang-format off
 #cmakedefine MIOPEN_DEFAULT_FIND_MODE @MIOPEN_DEFAULT_FIND_MODE@
+
 // clang-format on
-#cmakedefine MIOPEN_AMDGCN_ASSEMBLER "@MIOPEN_AMDGCN_ASSEMBLER@"
-#cmakedefine HIP_OC_COMPILER "@HIP_OC_COMPILER@"
-#cmakedefine MIOPEN_HIP_COMPILER "@MIOPEN_HIP_COMPILER@"
-#cmakedefine MIOPEN_OFFLOADBUNDLER_BIN "@MIOPEN_OFFLOADBUNDLER_BIN@"
+#ifdef MIOPEN_COMPILER_FLAGS_V2 /* Flag to switch to MIOPEN Compiler Flag Approach V2*/
```

Comment:
Why: see previous review comment.
```suggestion
#cmakedefine01 MIOPEN_USE_COMPILER_FLAGS_V2
#if MIOPEN_COMPILER_FLAGS_V2
```
[Notice] Comment is removed because it does not explain _why something is necessary_. It explains _what is being done_, which is the duplication of information.

Reply:
[resolved]